### ISSUE TITLE
quotesOnKeys prop doesn't seem to take effect

### ISSUE THREAD

**Issue by stevewillard** [mac-s-g/react-json-view#326]
When specifying `quotesOnKeys={false}` I still see quotes on the object:

<img width="151" alt="Screen Shot 2021-02-05 at 10 39 46 AM" src="https://user-images.githubusercontent.com/11420/107054732-804dad00-679e-11eb-895d-5f32059f77b0.png">

```javascript
 <ReactJson
      src={src}
      name={null}
      theme="monokai"
      displayObjectSize={false}
      indentWidth={2}
      iconStyle="triangle"
      displayDataTypes={false}
      enableClipboard={false}
      collapsed={collapsed}
      quotesOnKeys={false}
 />
```

**Comment by mac-s-g** [comment]
@stevewillard thanks for reporting this.
i confirmed i see the issue as well.  i see what's up.  the prop isn't fully implemented.  

quotesOnKeys is implemented for object names [here](https://github.com/mac-s-g/react-json-view/blob/1487e878232686f8d97fe4437dc8f60c564c8c37/src/js/components/ObjectName.js).  

but it needs to be implemented [here](https://github.com/mac-s-g/react-json-view/blob/1487e878232686f8d97fe4437dc8f60c564c8c37/src/js/components/VariableEditor.js#L91) as well.

i'll post an mr to fix.

**Comment by mac-s-g** [comment]
i will merge and publish to v 1.21.1 once tests all pass.

**Comment by mac-s-g** [comment]
fix published to 1.21.1.  thanks again for reporting the issue.

**Comment by stevewillard** [comment]
Thank you so much. I really appreciate such a quick turnaround!
